cinema/models.py

```diff
@@ -27,6 +27,10 @@ class Actor(models.Model):
     first_name = models.CharField(max_length=255)
     last_name = models.CharField(max_length=255)
 
+    @property
+    def full_name(self):
+        return self.first_name + " " + self.last_name
```

Comment:
i think f string better

Reply:
Yeah, I prefer it too, but just made in the same style as other code